schema/in_memory_entity/base.json

```diff
@@ -0,0 +1,24 @@
+{
+    "schemaId": "in-memory-entity/base",
```

Comment:
I was thinking about this in [update/SOF-6186](https://github.com/Exabyte-io/esse/tree/update/SOF-6186) PR - using this convention, it probably would be better to also have:

`property/raw.json`
`property/refined.json`

instead of

`properties/raw_property.json`